.github/workflows/publish.yml

```diff
@@ -0,0 +1,89 @@
+name: Publish package to npmjs
+
+# This workflow runs when code is pushed to `main` (i.e: when a pull request is merged)
+on:
+  push:
+    branches: [main]
+
+jobs:
+    version:
+        runs-on: ubuntu-latest
+
+        # OSBotify will update the version on `main`, so this check is important to prevent an infinite loop
+        if: ${{ github.actor != 'OSBotify' }}
+
+        steps:
+            # Running this action ensures that only one instance of this job will run at a time.
+            # This is important to prevent race conditions when multiple pull requests are merged in quick succession
+            - uses: softprops/turnstyle@8db075d65b19bf94e6e8687b504db69938dc3c65
```

Comment:
I think we're moving to GH's native `concurrency` feature, instead of using turnstyle. See https://github.com/Expensify/react-native-onyx/pull/252